dao/src/main/kotlin/tables/runs/shared/IssuesTable.kt

```diff
@@ -42,8 +43,19 @@ object IssuesTable : SortableTable("issues") {
 
 class IssueDao(id: EntityID<Long>) : LongEntity(id) {
     companion object : SortableEntityClass<IssueDao>(IssuesTable) {
+        fun findByIssue(issue: Issue): IssueDao? = find {
+            IssuesTable.issueSource eq issue.source and
+                    (IssuesTable.message eq issue.message) and
```

Comment:
Micro optimization: Check the `message` last as it is potentially the longest column

Reply:
From this declaration, an SQL statement is generated. I would expect that the database constructs its own optimized execution plan and that the change of the order does not have any effect.

Reply:
This is wishful thinking :)

Better explained:

> If none of the conditions apply, the order the conditions are specified may affect the order of evaluation, but since the database is going to have to fetch every single record to satisfy the query, the time spent fetching will likely dwarf the time spent evaluating the conditions.